src/multilspy/language_servers/pyright_language_server/pyright_server.py

```diff
@@ -0,0 +1,115 @@
+"""
+Provides Python specific instantiation of the LanguageServer class. Contains various configurations and settings specific to Python.
+"""
+
+import json
+import logging
+import os
+import pathlib
+from contextlib import asynccontextmanager
+from typing import AsyncIterator
+
+from multilspy.multilspy_logger import MultilspyLogger
+from multilspy.language_server import LanguageServer
+from multilspy.lsp_protocol_handler.server import ProcessLaunchInfo
+from multilspy.lsp_protocol_handler.lsp_types import InitializeParams
+from multilspy.multilspy_config import MultilspyConfig
+
+
+class PyRightServer(LanguageServer):
+    """
+    Provides Python specific instantiation of the LanguageServer class. Contains various configurations and settings specific to Python.
+    """
+
+    def __init__(self, config: MultilspyConfig, logger: MultilspyLogger, repository_root_path: str):
+        """
+        Creates a JediServer instance. This class is not meant to be instantiated directly. Use LanguageServer.create() instead.
+        """
+        super().__init__(
+            config,
+            logger,
+            repository_root_path,
+            ProcessLaunchInfo(cmd="basedpyright-langserver --stdio", cwd=repository_root_path),
```

Comment:
Could we please add at least a temporary setup_runtime_dependencies method, which for now just checks if "basedpyright-langserver" is installed, and if not, then links to a URL which guides how to install it, and fails gracefully with the error message